### PR TITLE
Ddocent

### PR DESCRIPTION
**Pull request by cmonjeau**
Hi,

dDocent must have vcftools v0.1.11 and the binary pearRM. Some update has been proposed in this pull request, in addition to dDocent.

### CODE REVIEW

File: recipes/pear/meta.yaml [hunk 1]
```diff
@@ -9,6 +9,8 @@ source:
   fn: pear-0.9.6-src.tar.gz
   md5: 5ed68d50c1620cd55c58681e38771799
   url: http://sco.h-its.org/exelixis/web/software/pear/files/pear-0.9.6-src.tar.gz
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/pear/meta.yaml]: I don't think you need to increase the version number if you only add tests.

File: recipes/ddocent/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - stacks
+    - trimmomatic
+    - mawk
+    - bwa
+    - samtools 0.1.19
+    - vcftools 0.1.11
+    - rainbow
+    - seqtk
+    - cd-hit
+    - bedtools 2.23.0
```
Comment (bgruening) [inline recipes/ddocent/meta.yaml]: This is an invalid syntax, isn't it? Do you need the strict dependencies or can you simply use bedtools. Same holds true for samtools and vcftools

Comment (cmonjeau) [inline recipes/ddocent/meta.yaml]: Hi. dDocent want specific versions of samtools, vcftools and bedtools. It explains why I've added bedtools and vcftools old releases recipes.

Comment (bgruening) [inline recipes/ddocent/meta.yaml]: But than you need to change this line to something like this: `- numpy >=1.5.1`

Comment (cmonjeau) [inline recipes/ddocent/meta.yaml]: Ok, I read the conda doc here : http://conda.pydata.org/docs/spec.html#build-version-spec

So you prefer:

 - bedtools ==2.23.0 ?.

Comment (bgruening) [inline recipes/ddocent/meta.yaml]: Yes, I guess this is what most of the recipes using.

**Comment by bgruening** [comment]
One more thing, but this can be in a separate PR. Please add all your tools to the os-x whitelist. So we also get OS-X builds for it.

**Pull request merged**